packages/stream_chat_flutter/lib/src/message_input/countdown_button.dart

```diff
@@ -1,36 +1,32 @@
 import 'package:flutter/material.dart';
 import 'package:stream_chat_flutter/stream_chat_flutter.dart';
 
-/// {@template countdownButton}
-/// Shows the countdown to when the user can send another message.
-/// {@endtemplate}
-class CountdownButton extends StatelessWidget {
-  /// {@macro countdownButton}
-  const CountdownButton({
+/// Button for showing visual component of slow mode.
+class StreamCountdownButton extends StatelessWidget {
+  /// Constructor for creating [StreamCountdownButton].
+  const StreamCountdownButton({
     super.key,
     required this.count,
   });
 
-  /// The countdown, in seconds.
+  /// Count of time remaining to show to the user.
   final int count;
 
   @override
-  Widget build(BuildContext context) {
-    return Padding(
-      padding: const EdgeInsets.all(8),
-      child: DecoratedBox(
-        decoration: BoxDecoration(
-          color: StreamChatTheme.of(context).colorTheme.disabled,
-          shape: BoxShape.circle,
-        ),
-        child: SizedBox(
-          height: 24,
-          width: 24,
-          child: Center(
-            child: Text('$count'),
+  Widget build(BuildContext context) => Padding(
```

Comment:
Why is this going back to using the arrow function?

Reply:
I'll revert it, but let's add a linter rule for this if we decided to not use it anymore

Reply:
I thought we did that already 🤔